### PR TITLE
docs(monitoring-system, gcf-utils): add documentation to READMEs for monitoring system components

### PR DESCRIPTION
**Pull request by azizsonawalla**
Adds documentation for:

* packages/monitoring-system
* packages/monitoring-system/data-processor
* packages/monitoring-system/metrics-dashboard

and updates documentation for:

* packages/gcf-utils

### CODE REVIEW

File: packages/gcf-utils/README.md [hunk 1]
```diff
@@ -29,6 +163,34 @@ npm install
 
 ## Testing
 
+### Run Unit Tests
+
+Run `npm run test` from the root directory to run the unit tests
+
+### Run Integration/System Tests
+
+Additional setup is required to run the `text/integration/gcf-bootstrapper-integration` tests. If you don't wish to run these tests, you can skip this setup. Else, follow these steps:
```
Comment (chingor13) [inline packages/gcf-utils/README.md:172]:
```suggestion
Additional setup is required to run the `test/integration/gcf-bootstrapper-integration` tests. If you don't wish to run these tests, you can skip this setup. Else, follow these steps:
```

File: packages/monitoring-system/README.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
-#  GitHub Automation Bots Monitoring System 
+#  GitHub Automation Bots Monitoring System (Internal Only)
 
-// TODO(asonawalla)
\ No newline at end of file
+A system to collect metrics on GitHub Repo Automation Bot executions and make it easily accessible to bot SREs and developers for debugging and facilitating the development process.
+
+[Read the design document here (internal)](http://go/automation-bot-monitoring-system-design)
+
+[Watch the presentation (starting 17m15s)](https://drive.google.com/file/d/1lmMbKiRUkE1BaCo4bR9lWKXJxOdaR01P/view&sa=D&ust=1598459279983000&usg=AFQjCNEA1FGlFyM4XtH2KaMgnvDjiLZT1g)
```
Comment (chingor13) [inline packages/monitoring-system/README.md:7]: Let's not link internal accessible links here

Comment (azizsonawalla) [reply packages/monitoring-system/README.md:7]: Is that for just this link or should I remove all internal links?

Comment (chingor13) [reply packages/monitoring-system/README.md:7]: Probably all of them.

**Pull request merged**